### ISSUE TITLE
Touch security key does not work

### ISSUE THREAD

**Issue by DGoiana** [zen-browser/desktop#437]
### What happened?

Touch ID security key does not work while attempting to login in gmail.

![Screenshot 2024-08-22 at 13 35 06](https://github.com/user-attachments/assets/4477d629-3ddb-4eb1-b9b3-3c5c080b42b1)

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.0-a.27

### Severity impact

Medium

### What platform are you seeing the problem on?

macOS - aarch64

**Comment by andmalc** [comment]
Same issue on a Thinkpad X280 running Fedora Linux 40 under Wayland.
Tried two Yubikeys: USB-A and USB-C. 
Zen installed from Flatpak.  
USB access is not the problem as Zen can see a USB flash drive and Flatseal shows device=all permission.
Firefox on this machine is able to access USB security keys fine.
Zen version: 1.0.0-a.27 (fresh install today)

**Comment by tancetiner** [comment]
It's the same issue with [this](https://github.com/zen-browser/desktop/issues/111). Looks like they closed it saying that maybe it's on the Firefox side but according to the last comment there it looks like it's not the case. I hope they look into that I'm having the same issue on every website with Passkey authentication.

**Comment by bengiv** [comment]
#111

**Comment by DGoiana** [comment]
I'll try looking into this

**Comment by elbe-vivek-c** [comment]
+1

**Comment by MaxNeedsSnacks** [comment]
> I just drop this here as it wasn't mentioned before. This might be related to not having this [entitlement](https://developer.apple.com/documentation/bundleresources/entitlements/com_apple_developer_web-browser_public-key-credential).
> 
> I'm not sure how easy it is to add that or if the app needs to pay the gatekeeper fee for proper signing.

You won't believe what comment I *just* left on the other issue because I forgot to read this one as well first 🤦

Either way, from what I can tell just paying the gatekeeper fee alone isn't gonna cut it, this is an **additional** entitlement unfortunately. I'm not sure how likely Apple is to give it out either, since they haven't done so for the other browsers I mentioned in the other issue (though that said, those browsers may not even have tried to apply either)

**Comment by aRustyDev** [comment]
I'm also having this issue, but its when I try to log back into my 1password plugin from Zen using the touch id instead of having to retype the password. 
I have the plugin in my firefox browser and its working just fine, so i don't think its a firefox side issue.

**Comment by twhitson** [comment]
I have 3 1Password accounts I log into separately but Touch ID manages that for me. Due to this bug, I can only use one of them at a time. This is unfortunately a blocker for me to use Zen Browser on my work computer, otherwise I love it.

**Comment by mauro-balades** [comment]
Update: macos has been signed! Let's wait for version 38 to build, and see if this issue goes away or 1password accepts zen or whatever

**Comment by alii** [comment]
Just to clarify @mauro-balades, what are we waiting on to get this resolved? I understood your comment above as "v38 will support passkeys," but I've tried the v39 alpha and it still doesn't seem to be working. If it's not been fixed due to a lack of human resources, is it something we could perhaps delegate to the community? I'd love to help out and expedite this if I can (for my own sake! 😄)

I understand this is OSS and your time is very precious, so thank you for your hard work.

**Comment by spojskic** [comment]
I can confirm that as of 24 September 2024, it is still not working on ARM macbooks.

**Comment by spojskic** [comment]
Version: 1.0.1-a.7 (64-bit)
Date: 2 Oct 2024
Issues: Passkey options is not rendered on some login pages and on Github I get this message: 
![image](https://github.com/user-attachments/assets/6b5ab535-97c4-4dbc-a839-9a4b26dd2d08)

On google it is still the same issue with the ARM macbook closed:
![image](https://github.com/user-attachments/assets/d904b8c7-da67-42b4-9143-b269f504e00a)

**Comment by qverkk** [comment]
Same issue for me, cannot use my yubikey as a 2fa. I can't even click the cancel button, it freezes after i click it

**Comment by mauro-balades** [comment]
What does firefox do?

**Comment by qverkk** [comment]
> What does firefox do?

Nvm, it's borked in firefox also, i remember it working a few updates back. Basically it let u click the 2fa key and it'd proceed to login, now it just hangs

**Comment by spojskic** [comment]
> > What does firefox do?
> 
> Nvm, it's borked in firefox also, i remember it working a few updates back. Basically it let u click the 2fa key and it'd proceed to login, now it just hangs

It lets me use passkey:
![image](https://github.com/user-attachments/assets/86c22287-220b-42f1-b642-ce2f0bbf0c8c)

(I didn't upload whole screenshot because of sensitive data and I am lazy to blur it)

**Comment by yodancristino** [comment]
- 1.0.1-a.9
- Macbook M-series
- 15 Oct 2024

Can confirm, passkey still wont work.

Still occurs even after doing the followin
- Complete removal and reinstallation of Zen
- Sync and unsync of account (i don't know why I did this XD)
- System reboot

**Comment by utopiaeh** [comment]
Still not work 
![shoot_2024-10-18 _15 43 Zen Browser](https://github.com/user-attachments/assets/c13a6c66-0ffa-49d5-a7bc-c7581cc7c458)

**Comment by pencilcheck** [comment]
yes, just tested on 1.0.1-a.12 (Firefox 131.0.3) (aarch64) still doesn't work (login google.com)

**Comment by robsonke** [comment]
And today was the first time my passkey with gmail worked!

**Comment by spojskic** [comment]
> And today was the first time my passkey with gmail worked!

It still doesn't work on: 1.0.1-a.17 (Firefox 132.0) (aarch64)
<img width="601" alt="image" src="https://github.com/user-attachments/assets/40559f3b-c67b-4522-95aa-dea6701ff720">

**Comment by logicfool** [comment]
> And today was the first time my passkey with gmail worked!

Does not work on mine not on gmail or any other website that requires passkey 

I've reinstalled zen twice by cleaning all its data and profiles

**Comment by thatcomputerguy0101** [comment]
I found that in [Firefox's production entitlements file](https://searchfox.org/mozilla-central/source/security/mac/hardenedruntime/v2/production/firefox.browser.xml), the following permissions are requested for passkeys. Based on Firefox's documentation, I believe this specific permission also requires something configured within the Apple Developer account used to sign the app.

```xml
<!-- Required for com.apple.developer.web-browser.public-key-credential -->
<key>com.apple.application-identifier</key>
<string>43AQ936H96.org.mozilla.firefox</string>

<!-- For platform passkey (webauthn) support -->
<key>com.apple.developer.web-browser.public-key-credential</key><true/>
```

The first of those keys seems like it will need edited once the necessary permission is approved.

**Comment by thatcomputerguy0101** [comment]
Looking through #111, I found [Apple's documentation on the entitlement required for browser Passkey support](https://developer.apple.com/documentation/bundleresources/entitlements/com_apple_developer_web-browser_public-key-credential). Seems like as a general web browser, Zen should qualify to be granted the entitlement as long as it being in alpha is not problematic.

**Comment by yodancristino** [comment]
10-November-2024 16:21 GMT+8

Passkeys functionality on Mac still not available.
1.0.1-a.18 (Firefox 132.0.1) (aarch64)

**Comment by bjsho** [comment]
> Looking through #111, I found [Apple's documentation on the entitlement required for browser Passkey support](https://developer.apple.com/documentation/bundleresources/entitlements/com_apple_developer_web-browser_public-key-credential). Seems like as a general web browser, Zen should qualify to be granted the entitlement as long as it being in alpha is not problematic.

I mean, Passkeys also work on Firefox Nightly, so I don't think that that it really a problem

**Comment by thatcomputerguy0101** [comment]
> I mean, Passkeys also work on Firefox Nightly, so I don't think that that it really a problem

Firefox's [Nightly Entitlements XML](https://searchfox.org/mozilla-central/source/security/mac/hardenedruntime/v2/production/nightly.browser.xml) still includes the same permission keys I said are needed for passkeys.

**Comment by pal9999** [comment]
Find this thread, it's so critical for me. Hope it'll be fixed

**Comment by yodancristino** [comment]
11-November-2024 23:04 GMT+8

Passkeys functionality on Mac still not available.
1.0.1-a.19 (Firefox 132.0.1) (aarch64)

**Comment by dennyzain** [comment]
1.0.1-a.19 Installed on: November 11, 2024
it still doesn't work on MacBook pro m2. I hope it'll fix ASAP, this critical feature nowadays

**Comment by monsieurgui** [comment]
> 1.0.1-a.19 Installed on: November 11, 2024 it still doesn't work on MacBook pro m2. I hope it'll fix ASAP, this critical feature nowadays

This a must needed feature, it should be prioritized I believe.

**Comment by pal9999** [comment]
Still not working. So critical for me(

**Comment by johann-taberlet** [comment]
I can confirm that the issue still exists on MBP M1 v15.1 and Zen v1.0.1-a.19.

**Comment by Kino8724** [comment]
Just installed zen and can confirm its not solved yet. This is for Version 1.0.1-a.19

**Comment by alii** [comment]
It's not particularly helpful to keep leaving comments about it not working.

We all know it's not working yet, and if something changes this thread will be updated. Watchers of this issue all get emails/notifications every time someone comments. You could leave a thumbs up reaction on the original post if you think this is a good idea 🙂

**Comment by obatola** [comment]
same here m1 mac

**Comment by iMonZ** [comment]
Issue still persist.
Passkeys do not work from 1Password or the iCloud Keychain.

**Comment by yodancristino** [comment]
Version: 1.0.2-b.0 (Firefox 133.0) (aarch64)
Macbook M-series
11 December 2024

Can confirm, passkey still wont work.

P.S. Congratulations on the beta release!

**Comment by MaxNeedsSnacks** [comment]
> I found that in [Firefox's production entitlements file](https://searchfox.org/mozilla-central/source/security/mac/hardenedruntime/v2/production/firefox.browser.xml), the following permissions are requested for passkeys. Based on Firefox's documentation, I believe this specific permission also requires something configured within the Apple Developer account used to sign the app.
> 
> ```
> 
> <key>com.apple.application-identifier</key>
> <string>43AQ936H96.org.mozilla.firefox</string>
> 
> 
> <key>com.apple.developer.web-browser.public-key-credential</key><true/>
> ```
> 
> The first of those keys seems like it will need edited once the necessary permission is approved.

For what it's worth since "it no worky" won't aid this conversation any further, it does seem like Apple isn't just artificially restricting this entitlement to the big guys like I originally feared. Orion, another smallish WebKit-based browser, recently [requested and obtained](https://orionfeedback.org/d/3025-support-passkeys/118) it themselves, so I guess it likely just needs to be requested by the person whose developer account is being used for codesigning at the moment.

**Comment by i3roly** [comment]
> > I found that in [Firefox's production entitlements file](https://searchfox.org/mozilla-central/source/security/mac/hardenedruntime/v2/production/firefox.browser.xml), the following permissions are requested for passkeys. Based on Firefox's documentation, I believe this specific permission also requires something configured within the Apple Developer account used to sign the app.
> > ```
> > 
> > <key>com.apple.application-identifier</key>
> > <string>43AQ936H96.org.mozilla.firefox</string>
> > 
> > 
> > <key>com.apple.developer.web-browser.public-key-credential</key><true/>
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > The first of those keys seems like it will need edited once the necessary permission is approved.
> 
> For what it's worth since "it no worky" won't aid this conversation any further, it does seem like Apple isn't just artificially restricting this entitlement to the big guys like I originally feared. Orion, another smallish WebKit-based browser, recently [requested and obtained](https://orionfeedback.org/d/3025-support-passkeys/118) it themselves, so I guess it likely just needs to be requested by the person whose developer account is being used for codesigning at the moment.

i agree. and fwiw i've used an official firefox build on my 10.15 machine and i get the same prompt as the others describe above.

i am pretty sure the QR-code based authentication is only available on newer versions of macOS . i think touch-based would work fine if i tried but i am not one of those people

**Comment by Eldevia** [comment]
For us it's also a deal breaker. Can't 2fa 90% of accounts!
Github, Google, Gitlab etc.

**Comment by mr-cheffy** [comment]
I think i've figured it out, we where missing the passkey (webauth) support entitlement (https://searchfox.org/mozilla-central/source/dom/webauthn/MacOSWebAuthnService.mm#506) enabled. I'll enable it for today's twilight, but we'll see because I need to register the app somewhere else, so it might take another day or 2.

edit: Woah, seems like y'all already figured out before haha... sorry for having this issue a bit abandoned. Ill make it my priority now

**Comment by mr-cheffy** [comment]
I also seem to be missing `com.apple.security.smartcard`, what a mess! Sorry, I dont have a mac so im kind of doing this blindly

**Comment by mr-cheffy** [comment]
Ok, sorry guys... Im removing the fix temporarily, because I cant have the permission until they accept this form, ill send it tomorrow first thing

![image](https://github.com/user-attachments/assets/26b263f6-2e8b-4359-8311-8ad357413953)

**Comment by rrooddooxx** [comment]
Hey @mr-cheff here a mac user! I'll be more than happy to help you on this! Let me know if I can assist

**Comment by mr-cheffy** [comment]
![image](https://github.com/user-attachments/assets/2914c0b7-09d6-4092-871d-d0906c9f2757)

LET'S GO! Ill push to twilight today and release tomorrow

**Comment by mr-cheffy** [comment]
https://github.com/zen-browser/desktop/pull/4095 hasn't been merged, im still trying to fix macos signing

**Comment by mr-cheffy** [comment]
Sorry guys... b6 wont be released with this fix... Ill still work day and night to get the signing correctly figured out. But I don't feel comfortable leaving people with a vulnerable firefox version, so ill have to release now.

**Comment by spojskic** [comment]
> Sorry guys... b6 wont be released with this fix... Ill still work day and night to get the signing correctly figured out. But I don't feel comfortable leaving people with a vulnerable firefox version, so ill have to release now.

No problem, you are doing a great job. We can wait. I am just happy that its being worked on

**Comment by mr-cheffy** [comment]
> No problem, you are doing a great job. We can wait. I am just happy that its being worked on

Yeah sorry, I was so close to finally figuring it out, but ill finish signing windows builds at 3am and I wont be home for the next 2 days, can't leave users to https://www.mozilla.org/en-US/security/advisories/mfsa2025-01/ for that long...

**Comment by apitamr** [comment]
still waiting for this

**Comment by naokichau** [comment]
this is the only road block left and I will switch from Arc to Zen

**Comment by CaioBianchi** [comment]
Same here. Been referred to this browser, and I'm loving the experience so far, but this is a huge blocker.

**Comment by PhilipMathieuIDEXX** [comment]
Looking forward to this becoming available. Thanks to the maintainers for making such a good project. Unfortunate that this isn’t something the community can contribute directly due to the permissions/signing aspect, but I get why that is the case.

From: Caio Bianchi ***@***.***>
Date: Thursday, January 23, 2025 at 12:04 AM
To: zen-browser/desktop ***@***.***>
Cc: Mathieu, Philip ***@***.***>, Manual ***@***.***>
Subject: Re: [zen-browser/desktop] Touch security key does not work (Issue #437)

Same here. Been referred to this browser, and I'm loving the experience so far, but this is a huge blocker.

**Comment by mr-cheffy** [comment]
Guys, I understand, but please don't write any comments that dont positively contribute to the issue "logically?". Because everyone subscribed gets a notification when you comment.

Im still working on this issue, you can follow the progress here:

https://github.com/zen-browser/desktop/pull/4601

I had to rewrite the codesign process so we now use Firefox's built in one, this will hopefully finally give us access to this feature

**Comment by nikelborm** [comment]
Hi guys!

I'm currently trying zen browser instead of chrome on Arch Linux with Gnome. In chrome it was possible to use my android phone as a pass key using face-recognition biometrics (the same one that can be used to unlock the phone itself). The browser usually connected to phone using Bluetooth after I selected my phone as device I want to use. Is this functionality available in zen-browser?

![Image](https://github.com/user-attachments/assets/6b1864b5-fb84-4593-a7d1-e0f08ad8ca19)

It just shows very little popup without any buttons and I don't know what am I supposed to touch, if I'm using face recognition + I don't see anywhere an option to use my phone as a device that stores passkeys, as I had in chrome.

**Comment by Rayrnond** [comment]
> Hi guys!
> 
> I'm currently trying zen browser instead of chrome. In chrome it was possible to use my android phone as a pass key using face-recognition biometrics (the same one that can be used to unlock the phone itself). The browser usually connected to phone using Bluetooth after I selected my phone as device I want to use. Is this functionality available in zen-browser?
> 
> ![Image](https://github.com/user-attachments/assets/6b1864b5-fb84-4593-a7d1-e0f08ad8ca19)
> 
> It just shows very little popup without any buttons and I don't know what am I supposed to touch, if I'm using face recognition + I don't see anywhere an option to use my phone as a device that stores passkeys, as I had in chrome.

If you're using macOS, this issue is going to make that option available. Apple requires apps to include an entitlement for that functionality.

**Comment by nikelborm** [comment]
No I'm not using mac os, I added the info that I'm using Linux 98 seconds after I wrote the message. Why disliking it immediately?

I'm just trying a new thing. I don't even know if some other popup supposed to be shown to select a device like my phone or not. And I don't know if it's a bug or there's just no such feature like using phone over Bluetooth as a passkey device.

UPD seems like there's just no such feature even in firefox 😿

**Comment by PhilipMathieuIDEXX** [comment]
@nikelborm This thread is specifically about macOS, so you would be better off posting a new issue (assuming there's not one specific to Linux yet). Maybe we should update the issue name to prevent confusion...

**Comment by dantaeusb** [comment]
You have reactions because there are probably around a hundred or a few hundred people subscribed to this waiting for constructive updates because it's important for them, and we all getting spammed almost daily with emails with zero constructive info on the topic, which makes whole subscription useless and annoying. It's just accumulated tiredness for over 50 comments essentially asking the same rhethoric "when" question, especially after developer response asking specifically to stop doing that and the OP problem is worked on. Yes, even if your question is different, it is stated in OP that it concerns a **specific** feature on a **specific** platform.

Please don't take it personally, and I would really ask everyone to consider whether their response contributes anything to the resolution of **this topic**.

**Comment by nikelborm** [comment]
I'm not asking when! I did't even know if it's a feature at all!

For the ones that actually want constructiveness I already found answer to my question. There's no such feature. It's not supported: [Bugzilla #1792433](https://bugzilla.mozilla.org/show_bug.cgi?id=1792433)

**Comment by mahdiarfrm** [comment]
is there a plan to fix this in any upcoming update?

**Comment by njimenezotto** [comment]
I think @mauro-balades is already working on resolving it. Check PRs #4601 and #4790 . I am waiting for the Twilight to test. Thanks @mauro-balades for the effort

**Comment by tsuliwaensis** [comment]
When will we see this released?

**Comment by franciscocabezas** [comment]
Just to add **+1** to the statistics

**Comment by yesseruser** [comment]
> Just to add **+1** to the statistics

I think this does nothing to help; the issue already has 112 thumbs ups and I think the maintainers are aware of it. Also, people are subscribed to this issue to see progress and getting spammed by comments like this.
This isn't targeted at your specific comment, but all "bump" comments.

**Comment by MR-UNI** [comment]
Passkeys still dont work? its been nearly a year! i remember they said it was a firefox side issue, but that is not the case. this seems like it should be a priority

**Comment by keramblock** [comment]
> Passkeys still dont work? its been nearly a year! i remember they said it was a firefox side issue, but that is not the case. this seems like it should be a priority

Talk is cheap, send patches

**Comment by Rayrnond** [comment]
> > Passkeys still dont work? its been nearly a year! i remember they said it was a firefox side issue, but that is not the case. this seems like it should be a priority
> 
> Talk is cheap, send patches

There isn't anything "we" can do since you need an Apple Developer account with the entitlement issued by Apple. The delay is outrageous, this should take an afternoon to add, not a year.

And Zen has the entitlement iirc.

**Comment by keramblock** [comment]
Last time I checked, Zen Browser was an open-source software provided to me and others for free. Nobody owns us anything here. If you want to change something, stop harassing developers and do something meaningful or move anywhere else.

**Comment by SuperManifolds** [comment]
> Last time I checked, Zen Browser was an open-source software provided to me and others for free. Nobody owns us anything here. If you want to change something, stop harassing developers and do something meaningful or move anywhere else.

In the comment above Rayrnond literally just explained why outside contributors cannot fix this, what is your problem?

**Comment by Froxcey** [comment]
# tldr: stop spamming
1. I don’t know the project maintainers’ intended use of GitHub issues is, but issues was originally designed as a bug tracker and not customer support, and most definitely not a forum. If you want to comment something, please make sure it directly helps with solving the issue itself. Useless messages are just wasting the maintainers’ limited time and attention. 
2. Whenever someone sends something here, EVERYONE subscribed to the issue hoping to get useful information gets a spam (like this message itself would too, and **I hope this is the last time it’s going to happen here**). Please be considerate and not spam everyone. Reach with a thumbs up on the original post should be enough. 
3. Idk what allows you to take feature requests for granted, especially when you’re using the product for free while the maintainer is out of their own pocket doing the project for passion. Only the project maintainer has a say on how they do their project, not you. Idk, being polite sounds like a basic and obvious manner to me.
4. It is really easy to find the pull request trying to solve this issue. Please at least spend minimal effort of searching before doing any of this. (And no, that doesn’t mean not having an open PR allows you to spam here)
5. I myself am not a maintainer and I’m not trying to act like one. I’m just writing what the general expectations are from GitHub issues that is somehow not immediately obvious to a bunch of ppl.

> We are tired of waiting for such a small but necessary feature. - rayrnond

I am more tired of spam messages (yours and many others’), and I think I’m unsubscribing from this thread.

Edit: Okay wtf… ppl really don’t know how to read or something it’s driving me insane…

**Comment by Rayrnond** [comment]
> # tldr: stop spamming
> 1. I don’t know the project maintainers’ intended use of GitHub issues is, but issues was originally designed as a bug tracker and not customer support, and most definitely not a forum. If you want to comment something, please make sure it directly helps with solving the issue itself. Useless messages are just wasting the maintainers’ limited time and attention. 
> 2. Whenever someone sends something here, EVERYONE subscribes to the issue hoping to get useful information gets a spam (like this message itself would too, and I hope this is the last time it’s going to happen here). Please be considerate and not spam everyone. Reach with a thumbs up on the original post should be enough. 
> 3. You shouldn’t take feature requests for granted, especially when you’re using the product for free while the maintainer is out of their own pocket doing the project for passion. Only the project maintainer has a say on how they do their project, not you. Idk, being polite sounds like a basic and obvious manner to me.
> 4. It is really easy to find the pull request trying to solve this issue. Please at least spend minimal effort of searching before doing any of this. (And no, that doesn’t mean not having an open PR allows you to spam here)
> 5. I myself am not a maintainer and I’m not trying to act like one. I’m just writing what the general expectations are from GitHub issues that is somehow not immediately obvious to a bunch of ppl.
> 
> > We are tired of waiting for such a small but necessary feature. - rayrnond
> 
> I am more tired of spam messages (yours and many others’), and I think I’m gonna unsubscribing to this thread.

Stop crying about a single email thread in your inbox. unsub if your sensitive, you yourself have now contributed to the aforementioned spam

**Comment by theoribeiro** [comment]
@mauro-balades or other maintainers, would you please lock this issue's comments? 

This conversation is absolutely pointless.

**Comment by mr-cheffy** [comment]
Successfully tested it and it works! It'll be on twilight from [this build](https://github.com/zen-browser/desktop/actions/runs/13874328458). Please do give it a try and let me know!

![Image](https://github.com/user-attachments/assets/b780edc0-560c-481d-9044-5795e79fb116)

**Comment by svencoeck** [comment]
Yes, I can confirm that it works perfectly. I was able to sign into Github using a passkey. 👍

**Comment by jakeleventhal** [comment]
Right when this releases/works, i get this problem https://github.com/zen-browser/desktop/issues/6315

after this i can finally switch over...

**Comment by nursandiid** [comment]
It works properly, I can't be patient to wait it

<img width="1920" alt="Image" src="https://github.com/user-attachments/assets/7b802d73-d6bf-4d1c-a218-a30b505c273d" />

**Comment by yodancristino** [comment]
I confirm that this now [works!](https://imgur.com/a/T1SsVwK)

**Comment by timtogan** [comment]
<img width="1912" alt="Image" src="https://github.com/user-attachments/assets/e98a396b-cdd1-4080-9f21-068af8e10829" /> Hi! For some reason passkeys do not work in my current profile, even after a reinstall of zen. They do however work in a new, fresh profile. Any way to make them work in my current profile?

**Comment by dphaldes** [comment]
I think this might have broken integration with password managers. My passkeys are stored in password manager (bitwarden specifically). Earlier, when a website prompted for a passkey, it would automatically trigger the password manager dialog for picking a key. Now, all it does it show a prompt for `Touch you security key` and doesnt trigger password manager at all. I am on Linux btw (flatpak - 1.10.1b (Firefox 136.0.2) (64-bit)). Not sure if this exact PR caused it

**Comment by mr-cheffy** [comment]
No, this issue is specifically about MacOS. Please open new issues if you are having similar problems. Thanks
